src/aai/aai.js

```diff
@@ -20,7 +20,7 @@
     this.setup = function(options) {
       var opts = jQuery.extend({}, this.defaults, options),
           defaultCallback = function(e) {
-            window.location = opts.host + '/Shibboleth.sso/Login?SAMLDS=1&target=' + opts.target + '&entityID=' + window.encodeURIComponent(e.entityID);
+            window.location = opts.host + '/Shibboleth.sso/Login?SAMLDS=1&target=' + opts.target + '?redirectUrl=' + window.location.href + '&entityID=' + window.encodeURIComponent(e.entityID);
```

Comment:
change it so it is more readable
target_url = opts.target + ?redirect...;
window.location = .... + target_url ?+ ....

Reply:
Done